### PR TITLE
add classList.add multiple arguments support in IE

### PR DESCRIPTION
**Pull request by v-zo**

### CODE REVIEW

File: src/services/IEService.ts [hunk 1]
```diff
@@ -2,8 +2,18 @@ import 'innersvg-polyfill';
...
 export default class IEService {
     static initialize() {
-        DOMTokenList.prototype.toggle = function (name, flag = !this.contains(name)) {
+        DOMTokenList.prototype.toggle = function(name, flag = !this.contains(name)) {
             return flag ? (this.add(name), true) : (this.remove(name), false);
         };
+
+        const oldAdd = DOMTokenList.prototype.add;
+        DOMTokenList.prototype.add = function() {
+            return [...arguments].map(name => oldAdd.call(this, name));
```
Comment (PavelKoroteev) [inline src/services/IEService.ts]: return void

File: src/services/IEService.ts [hunk 2]
```diff
@@ -2,8 +2,18 @@ import 'innersvg-polyfill';
 
 export default class IEService {
     static initialize() {
-        DOMTokenList.prototype.toggle = function (name, flag = !this.contains(name)) {
+        DOMTokenList.prototype.toggle = function(name, flag = !this.contains(name)) {
             return flag ? (this.add(name), true) : (this.remove(name), false);
         };
+
+        const oldAdd = DOMTokenList.prototype.add;
```
Comment (PavelKoroteev) [inline src/services/IEService.ts]: oldAdd ---> originAdd

**Review by PavelKoroteev — APPROVED** [review]

File: src/services/IEService.ts (hunk 2 above)
Comment (v-zo) [reply src/services/IEService.ts]: done

File: src/services/IEService.ts (hunk 1 above)
Comment (v-zo) [reply src/services/IEService.ts]: fixed

**Pull request merged**